packages/cmrjs/index.js

```diff
@@ -20,58 +20,38 @@ const logDetails = {
 };
 
 /**
- * Search for a concept on CMR
  *
- * @param {string} type - the concept type. Choices are: collection, granule
- * @param {Object} searchParams - cmr search params
+ * @param {string} type - Concept type to search, choices: ['collections', 'granules']
+ * @param {Object} searchParams - CMR search parameters
  * @param {Array} existingResults - array of results returned in previous recursive calls
- * @returns {Promise.<Array>} an array of search results
+ * @returns {Array} - array of search results.
  */
-async function searchConcept(type, searchParams, existingResults) {
-  const limit = process.env.CMR_LIMIT || 100;
-  const pageSize = process.env.CMR_PAGE_SIZE || 50;
+async function searchConcept(type, searchParams, existingResults = []) {
```

Comment:
We should probably put a test in for `searchConcept`

Reply:
Integration that hits CMR or unit test?  And what do you want it to test?

Reply:
unit test. We'd have to stub out the `got` call. The main thing would be nice to test the recursion and that the results are appended to the `existingResults` correctly.